homeassistant/components/awattar/config_flow.py

```diff
@@ -0,0 +1,47 @@
+"""Config flow for aWATTar integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.helpers.selector import selector
+
+from .const import CONF_COUNTRY, DOMAIN
+
+data_schema = {
+    vol.Required(CONF_COUNTRY): selector(
+        {
+            "select": {
+                "options": ["Austria", "Germany"],
```

Comment:
Checkout e.g. the `holiday` integration how to use the `SelectSelector`, this way we could make the options translatable